<?xml version="1.0"?>
<launch>

  <arg name="target_frame" default="velodyne"/>

  <!-- Transforms IMU tilted data in imu_link frame to ENU data in base_link frame -->
  <node pkg="nodelet" type="nodelet" name="imu_manager" args="manager"/>

  <!-- Transforms IMU tilted data in imu_link frame to ENU data in base_link frame -->
  <node pkg="nodelet" type="nodelet" name="imu_data_transformer" args="load imu_transformer/imu_transformer_nodelet imu_manager">
    <remap from="imu_in/data" to="/camera4/imu"/>
    <remap from="imu_out/data" to="/out"/>
    <param name="target_frame" value="$(arg target_frame)"/>
  </node>

</launch>
